### PR TITLE
Exclude rhumb from the build

### PR DESCRIPTION
**Pull request by mstade**
@grancalavera this oughta do it. When we merge this, we should also merge this: websdk/nap-examples#1.

### CODE REVIEW

**Comment by mstade** [comment]
_sigh..._ Of course the build fails because npm..

**Comment by mstade** [comment]
Heh, actuallly this is because in the makefile [we set the path to the local node_modules/.bin folder](https://github.com/websdk/nap/blob/exclude-rhumb/Makefile#L1). Node 5 ships with NPM 3, which dedupes and installs modules in an as-flat-as-possible manner, meaning somewhere deep in the dev dependencies there's something that depends on the [touch module](https://www.npmjs.com/package/touch). This then understandably gets confused because [we touch the node_modules dir](https://github.com/websdk/nap/blob/exclude-rhumb/Makefile#L7) after npm install, and that module doesn't really deal well with directories. (See: isaacs/node-touch/issues/8)

So yeah – the build fails because NPM, but I guess it's also pretty stupid to prepend `$PATH` instead of appending. Will fix.

**Comment by mstade** [comment]
That's strange, the build still fails with the same error – but it works fine locally with Node 5 and npm 3. Investigating..

**Comment by mstade** [comment]
Seems travis-ci/travis-ci#4862 covers what's going on, and offers a workaround. I'm going to implement the workaround for now, and then look into using wercker instead at some point down the line.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@websdk/nap",
-  "version": "0.9.8",
+  "version": "0.10.0",
```
Comment (grancalavera) [inline package.json:3]: Are you bumping the version by hand?

Comment (mstade) [inline package.json:3]: No, using `npm version`. I'm going with the no-one-really-cares-about-semver-before-1.0 approach, so bumping `minor` for the breaking changes and `patch` for non-breaking. Once we hit `1.0.0` though we should take better care.

Comment (grancalavera) [inline package.json:3]: cool

**Pull request merged**